src/import/trace-event.ts

```diff
@@ -65,16 +66,47 @@ interface XTraceEvent extends TraceEvent {
 // The trace format supports a number of event types that we ignore.
 type ImportableTraceEvent = BTraceEvent | ETraceEvent | XTraceEvent
 
+export interface StackFrame {
+  line: string
+  column: string
+  funcLine: string
+  funcColumn: string
+  name: string
+  category: string
+  // A parent function may or may not exist
+  parent?: number
+}
+
+export interface Sample {
+  cpu: string
+  name: string
+  ts: string
+  pid: number
+  tid: string
+  weight: string
+  // Will refer to an element in the stackFrames object
+  sf: number
+  stackFrameData?: StackFrame
+}
+
+export interface TraceEventJsonObject {
```

Comment:
`TraceEventWithSamples` to me sounds like it s a single `TraceEvent` with additional fields, instead of an object that contains an array of traceEvents, along with samples. How about `ChromeTraceWithSamples`?

Reply:
Ah, naming things is hard. `ChromeTraceWithSamples` sounds to me like it's specific to Chrome, and there's a whole separate importer in speedscope for Chrome profiles specifically (since they have their own format for CPU profiles that's totally unrelated to trace event -- the JavaScript profiler and the full-browser profiler are totally separate.

But I hear you that `TraceEventWithSamples` is also not a good name. `TraceEventsObjectWithSamples`?